plugin/src/main/scala/chisel3/internal/plugin/ChiselComponent.scala

```diff
@@ -79,12 +79,12 @@ class ChiselComponent(val global: Global, arguments: ChiselPluginArguments)
       }
     }
 
-    private val shouldMatchData: Type => Boolean = shouldMatchGen(tq"chisel3.Data")
+    private val shouldMatchData: Type => Boolean = shouldMatchGen(tq"chisel3.IsBindable")
```

Comment:
Can you rename this `val` and ensure anywhere else in this code that says `Data` is updated if it needs to be?

Reply:
ah yes, so far this patch has only renamed types, but i can also update vals, etc. that shouldn't really have "data" in their name anymore. i'll start with this one.